diff --git a/vllm_mindspore/__init__.py b/vllm_mindspore/__init__.py
index e7e3267..25dc071 100644
--- a/vllm_mindspore/__init__.py
+++ b/vllm_mindspore/__init__.py
@@ -241,7 +241,6 @@ from vllm_mindspore.multimodal.inputs import (as_kwargs, flat_build_elems,
                                               from_items, MultiModalFieldElem,
                                               _try_stack)
 
-from vllm.multimodal.inputs import MultiModalBatchedField
 from vllm.multimodal.inputs import MultiModalFlatField
 from vllm.multimodal.inputs import MultiModalKwargs
 
