diff --git a/vllm_mindspore/transformers_patch.py b/vllm_mindspore/transformers_patch.py
index ca8c7f8..234be8a 100644
--- a/vllm_mindspore/transformers_patch.py
+++ b/vllm_mindspore/transformers_patch.py
@@ -68,8 +68,10 @@ def patch_transformers():
     try:
         import mindone  # noqa: F401
     except ImportError:
-        warnings.warn("mindone.transformers not installed, "
-                      "skip patching transformers.", stacklevel=2)
+        warnings.warn(
+            "mindone.transformers not installed, "
+            "skip patching transformers.",
+            stacklevel=2)
         return
 
     import transformers
